RPIPs/RPIP-X.md

```diff
@@ -0,0 +1,308 @@
+
+##### Authorities
+
+* oDAO has the authority to veto changes approved by the pDAO if they have reason to believe the changes meet any of the following criteria according to [[RPIP-24](RPIP-24.md#odao-values)]:
+  * Malicious
+  * Not voted by the pDAO
+  * A result of vote manipulation in the pDAO
+  * Would result in clear damage to the Rocket Pool project
+* The oDAO has the authority to maintain its own membership. [[RPIP-24](RPIP-24.md#odao-values)]
+
+#### Guardian
+
+##### Description
+
+The guardian address is an EOA defined in the RocketStorage contract (but outside the key-value mappings.) It was responsible for the initialization of the protocol. Some of its initial powers have been irrevocably disabled, while others are still active at the time of writing.
+
+The guardian address is likely controlled by Rocket Pool Pty Ltd.
+
+##### Powers
+
+* The Guardian address has the power to modify pDAO settings via the pDAO settings pathway.
+* The Guardian address has the power to modify the share of RPL inflation via the pDAO bootstrap pathway.
+* The Guardian address has the power to spend the RPL tokens allocated to the pDAO via the pDAO bootstrap pathway.
+* The Guardian address has the power to set the maximum penalty rate for minipools via the RocketMinipoolPenalty contract.
+
+#### Authorities
+
+No authorities have been assigned to the guardian address by the pDAO. Indirect authorities are inherited from the assumption that the guardian is controlled by Rocket Pool Pty Ltd and are listed in that section.
+
+#### pDAO governance
+
+##### Description
+
+ProtocolDAO governance is defined as successful governance proposals voted on by Rocket Pool community members running Registered Nodes.
+
+##### Powers
+
+The pDAO currently has no direct, on-chain power over the Rocket Pool protocol.
+
+##### Authorities
+
+The pDAO as a collective is considered the principal of the Rocket Pool protocol. Authority over the Rocket Pool protocol lies with the pDAO, except that which relates to the oDAO, which maintains authority over itself. Changes to the protocol are only allowed to be done through following the pDAO's governance process.  
+
+Its authorities include, but are not limited to:
+
+* The authority to add, remove, replace, or amend RPIPs.
+* The authority to determine the membership of committees defined by RPIPs.
+* The authority to add, remove, or upgrade the functionality of the Rocket Pool protocol.
+* The authority to determine the distribution of RPL inflation.
+* The authority to engage agents on its behalf.
+* The authority to delegate specific authorities to separate governance processes.
+
+#### Individual Registered Nodes (oDAO or pDAO)
+
+##### Description
+
+Any individual Rocket Pool Registered Node. All Trusted Nodes (oDAO) are also Registered Nodes (pDAO).
```

Comment:
Minor technicality... A registered node running zero minipools has no pDAO vote power.

So it's more like "All pDAO members must have a Registered Node. All oDAO members must have a registered node, which is considered a Trusted Node."